buildtest/menu/get.py

```diff
@@ -33,9 +33,14 @@ def func_get_subcmd(args):
     root = os.path.join(BUILDSPEC_DEFAULT_PATH, "github.com")
     create_dir(root)
 
-    # Parse the repository name
+    # Parse the repository name assumes it is using HTTPS in format https://github.com/<username>/<repo>.git
     username = args.repo.split("/")[-2]
-    repo = args.repo.split("/")[-1]
+
+    # resolve url when its using SSH in format git@github.com:<username>/<repo>
+    if re.search("^(git@github.com)", args.repo):
+        url = args.repo.split(":")[1]
+        username = url.split("/")[0]
+
```

Comment:
Should there be any checks to see if something already exists at the clone path? Should the clone happen if this is the case?

Reply:
There is coverage when trying to reclone an existing repo that resolves to same path. See coverage report for get.py at https://coveralls.io/builds/30956049/source?filename=buildtest/menu/get.py

The check in framework looks fine, if dest directory exists we raise SystemError.

Reply:
The reclone test is found at https://github.com/shahzebsiddiqui/buildtest-1/blob/0b9bf6f1448df6c0def7fee848b38a84ba25d74a/tests/menu/test_get.py#L30